.gitlab-ci.yml

```diff
@@ -1,4 +1,4 @@
-include:
+ include:
```

Comment:
Why did you change this ? YAML is very sensitive to indentation changes, not sure this is what you want.

Reply:
probably was ktlint :(